Sources/Tracing/TracingClock.swift

```diff
@@ -0,0 +1,49 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Distributed Tracing open source project
+//
+// Copyright (c) 2020-2022 Apple Inc. and the Swift Distributed Tracing project
+// authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@available(macOS 13.0, iOS 15.0, *)
+public struct TracingClock: _Concurrency.Clock {
```

Comment:
We have to implement a TracingClock that gives us unit timestamps for "now", since there is no clock implementation in Swift which does this.

If we get one, we'd deprecate this type immediately